git_cml/git_utils.py

```diff
@@ -0,0 +1,118 @@
+import git
```

Comment:
I think in order to get this to work you'll need to make git_cml into a proper package, right? It was on my to-do list but I hadn't done it yet. Specifically you'll need to write an __init__.py file and have setup.py actually install the package.

Reply:
I'm not super familiar with packaging in python but the existing setup.py takes care of this already I think. At least when I do an editable install with `pip install -e <path to checkpoint-vcs>`, git_cml is a globally importable package (module?).

Reply:
In order for it to be pip-installable from Pypi (which we eventually want) we should add git_cml here:
https://github.com/r-three/checkpoint-vcs/blob/main/setup.py#L10
and in order to be able to access `git_cml.git_utils` by just doing `import git_cml` (instead of `from git_cml import blah` for every subpackage `blah`) we need to add a `__init__.py` file that imports all the subpackages like `from . import blah`.

Reply:
Gotcha, added in the latest commit